### ISSUE TITLE
Undefined: runtime.KeepAlive

### ISSUE THREAD

**Issue by quoidautre** [hanwen/go-mtpfs#131]
Hi,
I've an error doing this : 

```
$ sudo apt-get install golang fuse git-core libmtp-dev libfuse-dev
$ GOPATH=/tmp/go go get github.com/hanwen/go-mtpfs
# golang.org/x/sys/unix
/tmp/go/src/golang.org/x/sys/unix/ioctl.go:18: undefined: runtime.KeepAlive
/tmp/go/src/golang.org/x/sys/unix/ioctl.go:28: undefined: runtime.KeepAlive
```
I'm on ubuntu 16.04 : 
`Linux spectre 4.4.0-138-lowlatency #164-Ubuntu SMP PREEMPT Tue Oct 2 18:11:47 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux`

Have you an idea ?
Thanks
Fabrice

**Comment by Lin-Buo-Ren** [comment]
runtime.KeepAlive is introduced in Go 1.7, which is not available by the shipped Go version in Ubuntu 16.04

**Comment by quoidautre** [comment]
Hi,
Ok. Si, I've just waiting the new version... :)
